development.ini

```diff
@@ -1,11 +1,21 @@
 [server:main]
+# FOR DEVELOPMENT USE
 use = egg:Paste#http
 host = 0.0.0.0
 port = 8088
 
+# FOR PRODUCTION USE
+#use = egg:PasteScript#flup_fcgi_thread
+#socket = /opt/bcs/bcs-mumble/src/var/run/bcs-mumble.sock
+#umask = 0
+
 [app:main]
 use = egg:WebCore
+
+# FOR DEVELOPMENT USE
 debug = True
```

Comment:
For safety reasons,  please default this to False. Having this set to True allows remote code execution.